enhancements/two-node-fencing/tnf.md

```diff
@@ -205,6 +204,13 @@ An important facility of the installation flow is the transition from a CEO depl
 4. The installation proceeds as normal once the containers start. If for some reason, the etcd containers cannot be started, then the installation will fail. The installer will pull logs from the
 control-plane nodes to provide context for this failure.
 
+###### Managing Pacemaker and Resource/Fence agent Configuration
+Pacemaker configurations (as well as it's resource and fence agent configurations) do not need to be stored as files, as they are dynamically created using pcs commands. Instead, the in-cluster entity will handle triggering these commands as needed.
```

Comment:
The pcs commands do create files that are stored by the cluster though.
So any changes in subsequent releases will need to understand what the previous contents were